### ISSUE TITLE
View distribution

### ISSUE THREAD

**Issue by clintongormley** [zostay/CPAN-Sidekick#10]
At the moment I can view the documenation for a module, but it'd be great to be able to view the distribution too, so that I can choose to view other modules within the distro.

**Comment by zostay** [comment]
Planned but I hadn't added an issue for it yet. I do not know how soon this will happen. I am just one guy right now.

**Comment by clintongormley** [comment]
If you need some help with the queries, let me know
On Mar 10, 2012 4:47 PM, "Sterling Hanenkamp" <
reply@reply.github.com>
wrote:

**Comment by zostay** [comment]
This is now most of the way done, at least for phones. I have it setup so that you can tap on the module header in a POD document and get to the release. There will probably be menu items and a long press navigation option as well. Here's what it looks like.

![device-2012-12-13-210904](https://f.cloud.github.com/assets/34711/12524/cd63f08c-459b-11e2-9a4a-f740dac1fb12.png)

For some reason, Android hates your Gravatar, at least on the emulator. I'll have to check on a real device shortly, but I probably need a bug report for that here and see if there's something I can do to fix that as well.

Anyway, I hope to have the release viewer in the app before the year's out. We'll see if I can get it that polished before then.

**Comment by zostay** [comment]
I have an initial implementation of this screen that is working very nicely. I built the phone version first as seen above and then expanded the implementation for tablets. I now need to go back to the phone version and make sure I didn't break anything setting up the tablet version and do more testing, but I'm hopeful that I will have a release tomorrow with this in it.

Here's screenshot depicting the progress I've made on the tablet version. It's a bit nicer looking than the previous screenshot. The phone version will be similarly nicer looking.

![device-2012-12-16-005835](https://f.cloud.github.com/assets/34711/15375/0fe884aa-474e-11e2-9f6d-eef1949b57d6.png)

Also in the process I made the sidebar a bit wider here and on the search, which looks considerably better on my 10" tablet. I need to see how well it looks on a 7" tablet, but I suspect it will look good there too.
